### DISCUSSION TITLE
Can I use a sub-path when setting up using Link?

### DISCUSSION THREAD

**Discussion by mattpassarelli** [coder/code-server#3685] [Q&A]
I setup a new instance of code-server in a t2 EC2 instance in AWS. Works good so far, but I want make use of the EC2 instance for more than just code-server and want to put it on it's own sub-path or domain (code.mydomain.com or mydomain.com/code). As I understand this needs a reverse proxy from nginx or caddy. But is there something I can do to modify the Link setup that I had used instead? I don't have a problem setting up it again, and doing everything manually, but I do like using GitHub to authenticate and such.

Guide was here https://github.com/cdr/deploy-code-server/blob/main/guides/aws-ec2.md

**Comment by jsjoeio** [comment]
Hmm 🤔 I'm not sure if you can do that with Link, but @bpmct might know
